src/data/utils.ts

```diff
@@ -0,0 +1,5 @@
+import {Graph} from '../logic/commonInterfaces';
+
+export function adjMatrixToGraphString(matrix: Graph){
+  return matrix.map(row => row.join(' ')).join('\n');
```

Comment:
Single spaces are not really cutting it. We should use single spaces if there are only single-digit numbers in matrix. When there is even one number >9, function should start aligning graph with double spaces. Graph parsing handles that well.

Reply:
That still isn't algorithm that would align it correctly. Now that i looked at the samples we don't need it.